.github/workflows/upload-nightly-wheels.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         run: ls -lha dist/*.whl
 
       - name: Upload wheel to Anaconda Cloud as nightly
-        uses: scientific-python/upload-nightly-action@b67d7fcc0396e1128a474d1ab2b48aa94680f9fc # 0.5.0
+        uses: scientific-python/upload-nightly-action@ccf29c805b5d0c1dc31fa97fcdb962be074cade3 # 0.6.0
```

Comment:
```suggestion
        uses: scientific-python/upload-nightly-action@82396a2ed4269ba06c6b2988bb4fd568ef3c3d6b # 0.6.1
```

Reply:
Need v0.6.1 to avoid breakage. c.f. https://github.com/scientific-python/upload-nightly-action/issues/106